sql/core/src/test/resources/sql-tests/results/cte-nested.sql.out

```diff
@@ -64,10 +64,9 @@ WITH
   )
 SELECT * FROM t2
 -- !query schema
-struct<>
+struct<scalarsubquery():int>
 -- !query output
-org.apache.spark.sql.AnalysisException
-Name t is ambiguous in nested CTE. Please set spark.sql.legacy.ctePrecedencePolicy to CORRECTED so that name defined in inner CTE takes precedence. If set it to LEGACY, outer CTE definitions will take precedence. See more details in SPARK-28228.;
```

Comment:
I tried this query with Spark 2.4.5 (need to replace `t(c) AS (SELECT 1)` with `t AS (SELECT 1 c)` as Spark 2.4 doesn't support `t(c) syntax`), it fails the parser. So we don't need to fail here.

Reply:
Got it.